taxonomies/nutrients.txt

```diff
@@ -1938,49 +1938,96 @@ unit:en: mg
+fi: Kokonais-hiilihydraatit (sisältää kuitua), kokonais-hiilihydraatit, kokonais-hiilihydraatti
+fr: Glucides totaux (inclut les fibres), glucides totaux
+ga: Carbaihiodráit iomlán (cuimsíonn siúcra), carbaihiodráit iomlán
+he: פחמימות כוללות (כולל סיבים), פחמימות כוללות
+hr: Ukupni ugljikohidrati (uključuje vlakna), ukupni ugljikohidrati
+hu: Teljes szénhidrát (tartalmaz rostokat), teljes szénhidrát
+id: Total karbohidrat (termasuk serat), total karbohidrat
+it: Carboidrati totali (inclusi fibre), carboidrati totali
+ja: 総炭水化物（食物繊維を含む）、総炭水化物
+lt: Bendrieji angliavandeniai (įskaitant skaidulines medžiagas), bendrieji angliavandeniai
+lv: Kopējie ogļhidrāti (iekļauj šķiedru), kopējie ogļhidrāti
+mt: Karboidrati ġenerali (jinkludu l-fibra), karboidrati ġenerali
+nb: Totalt karbohydrat (inkluderer fiber), totalt karbohydrat
+nl: Totale koolhydraten (inclusief vezels), totale koolhydraten
+nl_be: Totale koolhydraten (inclusief vezels), totale koolhydraten
+pl: Całkowite węglowodany (zawiera błonnik), całkowite węglowodany
+pt: Carboidratos totais (inclui fibras), carboidratos totais, glicídios totais, glícidos totais, glucídios totais, glúcides totais, hidratos de carbono totais
+pt_br: Carboidratos totais (inclui fibras), carboidratos totais
+pt_pt: Hidratos de carbono totais (inclui fibras), hidratos de carbono totais, glícidos totais
+ro: Glucide totale (include fibre), glucide totale
+rs: Ukupni ugljeni hidrati (uključuje vlakna), ukupni ugljeni hidrati
+ru: Общие углеводы (включая клетчатку), общие углеводы
+sk: Celkové sacharidy (vrátane vlákniny), celkové sacharidy
+sl: Skupni ogljikovi hidrati (vključuje vlakna), skupni ogljikovi hidrati
+sv: Totalt kolhydrat (inkluderar fibrer), totalt kolhydrat
+tr: Toplam karbonhidratlar (lif içerir), toplam karbonhidratlar
+zh: 总碳水化合物（包括纤维），总碳水化合物
+zh_CN: 总碳水化合物（包括纤维），总碳水化合物
+zh_HK: 總碳水化合物（包括纖維），總碳水化合物
+zh_TW: 總碳水化合物（包括纖維），總碳水化合物
+unit:en: g
+
+# carbohydrates or net carbohydrates DO NOT include fiber
 zz: carbohydrates
-xx: Carbohydrates, carbohydrate
-en: Carbohydrates, carbohydrate
-ar: الكاربوهايدريد
-bg: Въглехидрати
-ca: Hidrats de carboni, Glúcids, Carbohidrats
-cs: Sacharidy
-da: Kulhydrat
-de: Kohlenhydrate
-el: Υδατάνθρακες
-es: Hidratos de carbono, Glúcidos, Carbohidratos
-et: Süsivesikud
-fa: کربوهیدرات ها
-fi: Hiilihydraatit, hiilihydraatti
-fr: Glucides
-ga: Carbaihiodráit
-he: פחמימות
-hr: Ugljikohidrati
-hu: Szénhidrát
-id: Karbohidrat
-it: Carboidrati
-ja: 炭水化物
-lt: Angliavandeniai
-lv: Ogļhidrāti
-mt: Karboidrati
-nb: Karbohydrat
-nl: Koolhydraten
-nl_be: Koolhydraten
-pl: Węglowodany
-pt: Carboidratos, Glicídios, Glícidos, Glucídios, Glúcides, Hidratos de carbono
-pt_br: Carboidratos
-pt_pt: Hidratos de carbono, Glícidos
-ro: Glucide
-rs: Ugljeni hidrati
-ru: Углеводы
-sk: Sacharidy
-sl: Ogljikove hidrate
-sv: Kolhydrat
-tr: Karbonhidratlar
-zh: 碳水化合物
-zh_CN: 碳水化合物
-zh_HK: 碳水化合物
-zh_TW: 碳水化合物
+xx: Carbohydrates (without fiber), Carbohydrates, carbohydrate
```

Comment:
I thought we had agreed not to change this yet, as at the moment (until we have done some kind of data migration) we don't really know if the `carbohydrates` nutrient includes fiber or not

Reply:
I'm going to approve with the name change for now but would be interested in others' opinions on this before we merge

Reply:
Contributors fixing US products from EU (through th data quality daily for example) might add the total carbohydrates in carbohydrates (see my last screen shot). Beside this, I am fine to switch back to carbohydrates since it indeed contains both with and without fibers currently.

Reply:
I'm a bit worried that showing "Carbohydrates (without fiber)" will be very confusing to all EU users scanning or adding EU products, as they will have to think if "Carbohydrates (without fiber)" is a different thing than the "Carbohydrates" field they see on the packaging.

I think it may be better to make things as simple as possible for the bulk of users and cases, and we can teach our contributors the difference between "Carbohydrates" and "Total carbohydrates".

Reply:
OK, I'll revert the taxonomy update that renames the current entry